client/my-sites/importer/importing-pane.jsx

```diff
@@ -115,15 +115,8 @@ export const ImportingPane = React.createClass( {
 
 	getHeadingText: function() {
 		return this.translate(
-			'Importing may take a while, but you can ' +
-			'safely navigate away from this page if you need ' +
-			'to. If you {{b}}stop the import{{/b}}, your site ' +
-			'will be {{b2}}partially imported{{/b2}}.', {
-				components: {
-					b: <strong />,
-					b2: <strong />
-				}
-			}
+			'Importing should take 15 minutes, sometimes more if your site has a lot of media. ' +
```

Comment:
@roundhill that comma doesn't belong there. How about something like this…

> The import process requires at least fifteen minutes to complete but may take longer depending on how many media items are in the site.

…maybe we can get some editorial review?